database/frontend/online-code-editors.json

```diff
@@ -27,6 +27,13 @@
     "category": "frontend",
     "subcategory": "online-code-editors"
   },
+  {
+    "name": "StackEdit",
+    "description": "StackEdit is a user-friendly online markdown editor with real-time collaboration and cloud storage integration, making writing and editing documents effortless and accessible from anywhere.",
+    "url": "https://stackedit.io/",
+    "category": "frontend",
+    "subcategory": "online-code-editors"
+  },
   {
     "name": "Sololearn",
     "description": "It's multi-language online code editor test and share your code snippets.",
```

Comment:
[Proofreading]
``suggestion
    "description": "It's multi-language online code editor where you can test and share your code snippets.",
```

Reply:
It done